scripts/release.ts

```diff
@@ -21,19 +21,32 @@ const verboseExec = (cmd: string) => {
 };
 
 const checkPrerequisites = () => {
+  const minHubVersion = "2.14.1";
+  let result;
+
   try {
-    execSync("git --version", {
-      stdio: "ignore",
-    });
-    execSync("hub --version", {
-      stdio: "ignore",
-    });
+    result = execSync("hub --version", {
+      stdio: "pipe",
+    }).toString("utf-8");
   } catch {
-    throw new Error(`
-  Please install missing dependencies:
-    - https://git-scm.com
-    - https://github.com/github/hub
-`);
+    console.log(
+      `\nCould not find the GitHub CLI tool >= ${minHubVersion}.\n` +
+        "You can get it here:\n\n" +
+        "  https://github.com/github/hub\n"
+    );
+    process.exit(1);
+  }
+
+  const match = result.match("hub version (.*)");
+
+  if (match) {
```

Comment:
We should also throw when we can’t extract or parse the versoin.

Reply:
Good catch!

https://github.com/radicle-dev/radicle-upstream/pull/1601/commits/93b4b53731bfc9f30e05e785e51e15816afa6a4d